pages/app-deploy/static/tips.js

```diff
@@ -154,5 +157,33 @@ location ~* \\.(?:css|js|otf|ttf|eot|woff|woff2)$ {
   add_header Cache-Control "public";
 }`}
     </Highlight>
+
+    <h3 id="copy-after-build">انتقال خودکار فایل به مسیر build پروژه</h3>
+
+    <p>
+      درصورتی که قصد داشته باشید پس از build شدن پروژه، فایلی را به‌صورت خودکار
+      به مسیر build انتقال دهید می‌توانید اسکریپت{" "}
+      <span className="code">build</span> پروژه‌ی خود را در فایل{" "}
+      <span className="code">package.json</span> به شکل زیر ویرایش کنید. در
+      مثال‌های زیر، فایل <span className="code">liara_nginx.conf</span> به‌صورت
+      خودکار از مسیر اصلی پروژه در پوشه‌ی <span className="code">build</span>{" "}
+      کپی می‌شود.
+    </p>
+
+    <h4>ویندوز:</h4>
+    <Highlight className="json">
+      {`"scripts": {
+  ...
+  "build": "react-scripts build && copy liara_nginx.conf ./build/",
```

Comment:
We have Linux servers. Does this work on our servers too?

Reply:
Users should build their app on local machine.